.vscode/settings.json

```diff
@@ -4,8 +4,8 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "typescript.enablePromptUseWorkspaceTsdk": true,
   "editor.codeActionsOnSave": {
-    "quickfix.rome": true,

```

Comment:
do we want this changed?

Reply:
Yeah, I just didn't want that change tucked into an outside contributors changeset.